tutorials/workflow/java/workflow-management/src/main/java/io/dapr/springboot/examples/WorkflowManagementRestController.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2025 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.springboot.examples;
+
+
+import io.dapr.spring.workflows.config.EnableDaprWorkflows;
+import io.dapr.springboot.examples.mgmt.NeverEndingWorkflow;
+import io.dapr.workflows.client.DaprWorkflowClient;
+import io.dapr.workflows.client.WorkflowInstanceStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.concurrent.TimeoutException;
+
+@RestController
```

Comment:
I don't see the private field of `instanceId` here as with the other examples, is this intentional?

Reply:
@marcduiker for this one is not needed as in your example requests you were sending the ID for each request.. so I decided to follow that.. do you want me to align this one with the private field?